sources/packages/backend/apps/api/src/route-controllers/models/pagination.dto.ts

```diff
@@ -101,3 +101,12 @@ export class OfferingChangePaginationOptionsAPIInDTO extends PaginationOptionsAP
   @IsIn(["applicationNumber", "fullName"])
   sortField?: string;
 }
+
+/**
+ * Extended pagination option for the inprogress and completed application offering change.
+ */
+export class OfferingChangePaginationRequestedOptionsAPIInDTO extends PaginationOptionsAPIInDTO {
```

Comment:
I see that OfferingChangePaginationRequestedOptionsAPIInDTO is shared between both API end points in-progress and completed. But we decided in previous PR to go with individual pagination DTO for an endpoint right?
Let me know if I am missing something.

Reply:
here the service is same for both endpoints that use extended pagination dto. Do we need to create separated extended pagination DTO, even if the service is the same?

Reply:
In the previous conversation we were talking about the target endpoint precisely.  Going by that, we can create seperate DTOs. 
![image](https://github.com/bcgov/SIMS/assets/54600590/57f38072-3638-45bd-a212-2b86ef614655)

Reply:
As discussed, we can ask other devs opinion and adjust if required in next PR.